src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int ELEMENT_PREVIOUSLY_COMMA = 0;
+    private static final int ELEMENT_AFTER_COMMA = 1;
+    private static final String REGEX = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName, true))) {
+            bufferedWriter.write(writeStatistic(fromFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write file", e);
+        }
```

Comment:
Create a separate method to write to the file, and call your methods here in turn